app/footer.tsx

```diff
@@ -1,3 +1,178 @@
-export default function Footer() {
-  return <></>;
+import Image from 'next/image';
+import footerimg from '../public/assets/footerimg.png';
+import { FaLinkedinIn } from 'react-icons/fa';
+import { FaFacebookSquare } from 'react-icons/fa';
+import { FaTwitter } from 'react-icons/fa';
+import { MdOutlineCall } from 'react-icons/md';
+import { MdMailOutline } from 'react-icons/md';
+import { CiGlobe } from 'react-icons/ci';
+import { SlLocationPin } from 'react-icons/sl';
+import logo from '../public/assets/nitkkrlogo.png';
+
+const quickLinks = [
+  {
+    text: 'Telephone Directory',
+    link: '',
+  },
+  {
+    text: 'Books and e-library',
+    link: '',
+  },
+  {
+    text: 'Important Links',
+    link: '',
+  },
+  {
+    text: 'Downloads',
+    link: '',
+  },
+  {
+    text: 'RTI',
+    link: '',
+  },
+  {
+    text: 'NIT KKR IT Infrastructure Usage Policy',
+    link: '',
+  },
+  {
+    text: 'NIT KKR @NDL',
+    link: '',
+  },
+  {
+    text: 'Telephone Directory',
+    link: '',
+  },
+];
+
+const aboutUsItems = [
+  { text: 'Council of NITs', link: '' },
+  { text: 'NAD Digilocker', link: '' },
+  { text: 'NIRF (Data & Certificate)', link: '' },
+  { text: 'NBA Accreditation Status', link: '' },
+  { text: 'ARIIA', link: '' },
+  { text: 'Skill Hub (PMKVY 4.0)', link: '' },
+  { text: 'Jobs @ NIT KKR', link: '' },
+];
+
+const departmentsItems = [
+  { text: 'Vigilance Corner', link: '' },
+  { text: 'PRO', link: '' },
+  { text: 'OBC & PWD Cell', link: '' },
+  { text: 'SC/ST Cell', link: '' },
+  { text: 'Join Alumni Association', link: '' },
+  { text: 'User Login', link: '' },
+  { text: 'Contact Us', link: '' },
+];
+
+export default function footer() {
+  return (
+    <>
+      <div className="flex md:flex-row lg:items-center justify-center flex-col">
+        <div className="flex flex-col items-center justify-center items px-10">
+          <Image src={logo} alt="" className="h-[66px] w-[66px]" />
+          <div className="font-display text-[22px] pt-3">
+            National Institute of
+          </div>
+          <div className="font-display text-[22px]">
+            Technology, Kurukshetra
+          </div>
+          <div className="font-sans text-[16px] text-[#8B949E] pt-3">
+            WRXF+23G, NIT, Thanesar, Haryana
+          </div>
+          <div className="font-sans text-[16px] text-[#8B949E]">136119</div>
```

Comment:
This is not how you divide a text into multiple lines. Put it in a single div and apply a max width